rpcpb/rpc.proto

```diff
@@ -141,6 +141,7 @@ message NodeInfo {
   string plugin_dir           = 7;
   string whitelisted_subnets  = 8;
   bytes config                = 9;
+  bool alive                  = 10;
```

Comment:
Do we really want this? As this is over gRPC, this can generate tricky situations because the client receives `Alive` when the state can be quickly changing on the actual process.

I feel like this is introducing potentially dangerous stuff.

Reply:
I think it is ok to notify the user on dead node process and let him dedice what to do. Really a process starts alive and is set to not being alive when received the channel notification